playbooks/roles/mattermost/templates/config.json.j2

```diff
@@ -164,38 +186,37 @@
         "EnableSignInWithEmail": true,
         "EnableSignInWithUsername": true,
         "SendEmailNotifications": true,
-        "UseChannelInEmailNotifications": false,
-        "RequireEmailVerification": true,
-        "FeedbackName": "Mattermost Notification",
+        "RequireEmailVerification": {{ MATTERMOST_REQUIRE_EMAIL_VERIFICATION }},
+        "FeedbackName": "{{ MATTERMOST_FEEDBACK_NAME }}",
         "FeedbackEmail": "{{ MATTERMOST_FEEDBACK_EMAIL }}",
         "FeedbackOrganization": "",
-        "EnableSMTPAuth": false,
-        "SMTPUsername": "",
-        "SMTPPassword": "",
-        "SMTPServer": "localhost",
-        "SMTPPort": "25",
-        "ConnectionSecurity": "",
+        "SMTPUsername": "{{ MATTERMOST_SMTP_USERNAME }}",
+        "SMTPPassword": "{{ MATTERMOST_SMTP_PASSWORD }}",
+        "SMTPServer": "{{ MATTERMOST_SMTP_SERVER }}",
+        "SMTPPort": "{{ MATTERMOST_SMTP_PORT }}",
+        "ConnectionSecurity": "TLS",
```

Comment:
`ConnectionSecurity` should probably also be behind a variable.

Reply:
Done.